flexibleengine/acceptance/acceptance.go

```diff
@@ -128,3 +128,11 @@ func testAccPreCheckSms(t *testing.T) {
 		t.Skip("OS_SMS_SOURCE_SERVER must be set for SMS acceptance tests")
 	}
 }
+
+func testAccPreCheckS3(t *testing.T) {
```

Comment:
```suggestion
func testAccPreCheckOBS(t *testing.T) {
```

it's better to use `OBS` instaed of `S3`